src/openzaak/conf/includes/base.py

```diff
@@ -693,6 +693,13 @@
     "mozilla_django_oidc_db.backends.OIDCAuthenticationBackend",
 ]
 
+ENABLE_2FA = config("ENABLE_2FA", default=False)
+
+# if ENABLE_2FA is true, fill the MAYKIN_2FA_ALLOW_MFA_BYPASS_BACKENDS with all
+# configured AUTHENTICATION_BACKENDS and thus disabeling the entire 2FA chain
+if ENABLE_2FA:  # pragma: no cover
```

Comment:
I don't get it, why do you disable 2FA with `ENABLE_2FA` env var? 
Could you just move this piece of code from `conf/dev.py` to `conf/includes/base.py`?